scripts/update_PR_comment.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+  # the text of the message that will replace the current one
+  message="${1}" #=$(python ./scripts/import-graph-report.py base.json head.json changed_files.txt)
```

Comment:
I guess this comment can now be removed?
```suggestion
  message="${1}"
```